README.md

```diff
@@ -18,3 +18,16 @@ The CSV files exported are designed to be used with [LosslessCut](https://github
 - ^ again, with camera recordings + offset CSV
 
 This workflow takes me ~3 minutes, and it enables my team to quickly grab 5+ videos worth of content after every stream.
+
+## Dev Setup:
+
+- First you will need a project setup in clerk. 
+  - Create a new project and grab your public + secret keys from the API Keys page.
+  - Copy them.
+  - Save them in a .env.local file.
+- Next you will need a twitch application (https://dev.twitch.tv/docs/authentication/)
+  - Paste them in to your .env.local under `TWITCH_CLIENT_SECRET` + `TWITCH_CLIENT_ID`
+- Install the node modules (make sure you are on node 16+)
+  (if you need to work with multiple versions of node then I highly recommend nvm!)
+- Start up the app with `npm run dev`
+- And presto. Should have running next app!
```

Comment:
### Create Your .env File
1) Set up the project with [Clerk](https://clerk.com/).
```suggestion
- First you will need a project setup in clerk. 
  - Create a new project and grab your public + secret keys from the API Keys page.
  - Copy them.
  - Save them in a .env.local file.
- Next you will need a twitch application (https://dev.twitch.tv/docs/authentication/)
  - Paste them in to your .env.local under `TWITCH_CLIENT_SECRET` + `TWITCH_CLIENT_ID`
- Install the node modules (make sure you are on node 16+)
  (if you need to work with multiple versions of node then I highly recommend nvm!)
- Start up the app with `npm run dev`
- And presto. Should have running next app!
```
    - Create a new project
    - Copy your Public and Secret keys and save them in a `.env.local` file.
    - For troubleshooting and more information on how to setup Clerk, visit their documentation [here](https://clerk.com/docs).
2) Next, you will need a [Twitch](https://dev.twitch.tv/docs/authentication/) application
    - Copy your Twitch application ID and Secret Key into your `.env.local`
### Run Locally
1) Install dependencies 
    - `npm install`
2) Run in development
    - `npm run dev`

Reply:
I think that you have to use a custom Clerk OAuth credential for Twitch as well to get the `user:broadcast:read` permission